spec.md

```diff
@@ -227,7 +227,7 @@ This object MAY be extended with [Specification Extensions](#specification-exten
 
 ### Example Pairing Object
 
-The Example Pairing object consists of a set of example params and result. The result is what you can expect from the JSON-RPC service given the exact params.
+The Example Pairing object consists of a set of example params and result. The result is what you can expect from the JSON-RPC service given the exact params. JSON-Schema examples MAY be used as a fallback for Example Pairings.
```

Comment:
```suggestion
The Example Pairing object consists of a set of example params and result. The result is what you can expect from the JSON-RPC service given the exact params. _JSON Schema_ examples MAY be used as a fallback for Example Pairings.
```

Reply:
Dunno if italic is gg, but

Reply:
going to keep it consistent and use `JSON Schema` no dash, and no italic. made a suggestion below to reflect that

Reply:
Yup, commited https://github.com/open-rpc/spec/pull/323/commits/ae02ddd6e0c6a48cdf220bf7adeae589351b2f86